src/GLMM.jl

```diff
@@ -92,7 +91,7 @@ end
 
 
 # X₀: check if it includes intercept
-function SuSiEGLMM(L::Int64,Π::Vector{Float64},yt::Vector{Float64},Xt::Matrix{Float64},Xt₀::Matrix{Float64},
+function susieGLMM(L::Int64,Π::Vector{Float64},yt::Vector{Float64},Xt::Matrix{Float64},Xt₀::Matrix{Float64},
```

Comment:
Looks very good -- this function seems to be working? We can set default L = 10 as in susieR

Reply:
Need to fix loco version, then it will work.